connectors/models.py

```diff
@@ -95,13 +95,14 @@
     Source.GRAFANA: [
         [
             SourceKeyType.GRAFANA_HOST,
-            SourceKeyType.GRAFANA_API_KEY
+            SourceKeyType.GRAFANA_API_KEY,
+            SourceKeyType.SSL_VERIFY,
```

Comment:
Create another entry with just this combination: 
[
            SourceKeyType.GRAFANA_HOST,
            SourceKeyType.GRAFANA_API_KEY
]

Reply:
You havent added an entry for GRAFANA_MIMIR?